src/action/payment.js

```diff
@@ -321,7 +363,8 @@ class PaymentAction {
   }
 
   /**
-   * Send the amount specified in the invoice as a lightning transaction and
+   * Send the amount specified in the payment.amount. If zero use the amount
```

Comment:
Wonky description here: `If zero use the amount specified in the invoice as a lightning transaction and display the wait screen while the payment confirms.` doesn't really make sense.

Reply:
Will try to make it more clear.

Reply:
Fixed in 55985e7d